MIGRATION.md

```diff
@@ -0,0 +1,155 @@
+apppool_idle_timeout_action      | idle_timeout_action
+
+### `iis_site`
+This type is still called `iis_site`
+
+The sslFlags attribute was added to specify the binding used for Secure Sockets
+Layer certificates.
+https://www.iis.net/configreference/system.applicationhost/sites/site/bindings/binding
+
+sslFlags 	Optional uint attribute.
+
+Specifies the type of binding used for Secure Sockets Layer (SSL) certificates.
+
+- A value of "0" specifies that the secure connection be made using an IP/Port
+  combination. Only one certificate can be bound to a combination of IP address
+  and the port.
+- A value of "1" specifies that the secure connection be made using the port
+  number and the host name obtained by using Server Name Indication (SNI).
+- A value of "2" specifies that the secure connection be made using the
+  centralized SSL certificate store without requiring a Server Name Indicator.
+- A value of "3" specifies that the secure connection be made using the
+  centralized SSL certificate store while requiring Server Name Indicator
+
+Centralized SSL certificate support enables you to create a centralized
+certificate store that can contain multiple certificate files. You can name the
+certificate files to correspond to the host names that they contain. This
+enables you to create a binding that only requires a port, rather than an
+IP/port or a host name/port combination. When a request comes in, IIS matches
+the port, determines the host name from the request, and searches the
+centralized certificate store for a certificate file with a matching name. It
+uses that certificate.
+
+With a Server Name Indicator (SNI), the host name is exchanged as part of the
+SSL handshake. SNI is enabled in the Add Site Binding dialog box when you add a
+binding with a type of HTTPS. This is especially useful for SSL connections
+that host multiple servers on a single network address. For more information,
+see IIS 8.0 Server Name Indication (SNI): SSL Scalability.
+
+The sslFlags attribute is only set when the protocol is https.
+
+The default value is 0.
+#### Parameter mapping:
+puppet-iis  | puppetlabs-iis
+------------|--------------
+ensure      | identical
+name        | identical
+path        | physicalpath
+app_pool    | applicationpool
+host_header | bindings hostname*
+protocol    | bindings protocol*
+ip          | bindings bindinginformation*
+port        | bindings bindinginformation*
+ssl         | bindings sslflags*
+
+\* The `bindings` parameter on the new `iis_site` take an array of one or more bindings as a hash and have the following keys per hash:
+- protocol
+- bindinginformation
+- sslflags
+
+The bindinginformation is in the form `<ip>:<port>`
+
+### `iis::manage_binding`
+All of these properties are managed directly by the `iis_site::bindings` property and are not declared separately, as the `iis_site` and `iis::manage_binding` resources are. The following describe the `iis_site::bindings` hash keys for the mapping:
```

Comment:
Verb agreement problem in the last sentence; for better readability, I suggest changing to:

The `iis_site::bindings` hash keys for the map are as follows: